lib/fabrik/DirectorManager.js

```diff
@@ -566,17 +566,19 @@ class DirectorManager extends BaseManager {
         secret: undefined,
         agent_ip: undefined,
         started_at: backupStartedAt,
-        finished_at: null
+        finished_at: null,
+        tenant_guid: this.getTenantGuid(opts.context)
```

Comment:
as part of this implementation then are we dropping `org_guid`? i was thinking in case of CF the tenant_guid would be `orgGuid_spaceGuid`. Is it a good idea to drop org guids?

Reply:
Org_guid wasn't part of the tenant ID in case of backup metadata file. I kept it similar, as otherwise there will be migration effort. technically, spaceguid is unique for each space so IMHO, it serves the same purpose. Do you see a problem in this?